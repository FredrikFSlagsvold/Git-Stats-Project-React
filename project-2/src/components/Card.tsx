type CardProps = {
    authorName: string
    title: string
    committedDate: String
}

export default function Card( {authorName, title, committedDate}: CardProps){
return (
<div className="commitsBox">
    <p className="text-xl"> {title}</p>
    <p>Author: {authorName}</p>
    <p>Commeted at: {committedDate.substring(0,10)}</p>

</div>)
}