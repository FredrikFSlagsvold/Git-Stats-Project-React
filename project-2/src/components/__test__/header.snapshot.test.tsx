import React, {useState} from 'react';
import renderer from 'react-test-renderer';
import Card from '../Card';


it('renders correctly', () => {
  const tree = renderer
    .create(<Card authorName={''} title={''} committedDate={""} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});

