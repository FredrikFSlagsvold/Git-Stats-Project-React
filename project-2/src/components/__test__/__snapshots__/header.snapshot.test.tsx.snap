// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="commitsBox"
>
  <p
    className="text-xl"
  >
     
  </p>
  <p>
    Author: 
  </p>
  <p>
    Commeted at: 
  </p>
</div>
`;
